alibi_detect/saving/schemas.py

```diff
@@ -889,6 +867,314 @@ class LearnedKernelDriftConfigResolved(DriftDetectorConfigResolved):
     train_kwargs: Optional[dict] = None
     dataset: Optional[Callable] = None
     device: Optional[Literal['cpu', 'cuda']] = None
+    dataloader: Optional[Callable] = None  # TODO: placeholder, will need to be updated for pytorch implementation
+
+
+class ContextMMDDriftConfig(DriftDetectorConfig):
+    """
+    Unresolved schema for the
```

Comment:
Remove "unresolved" since you've done it for others..?

Reply:
Detectors are different, as they do have an unresolved and resolved pydantic model.